src/nupic/frameworks/opf/common_models/cluster_params.py

```diff
@@ -29,14 +29,17 @@
 def getScalarMetricWithTimeOfDayAnomalyParams(metricData,
                                               minVal=None,
                                               maxVal=None,
-                                              minResolution=None):
+                                              minResolution=None,
+                                              tmImplementation = "cpp"):
```

Comment:
Shouldn't this be `tm_cpp` to match `temporalImp`?  (There are multiple CPP implementations, so 'cpp' is ambiguous.)